userguide/dailyuse/anbox.rst

```diff
@@ -1,13 +1,15 @@
 Android apps
 ========================
 
-`Anbox <https://anbox.io>`_ is a minimal Android container and compatibility layer that allows you to run Android apps on GNU/Linux operating systems such as UBports.
+`Anbox <https://anbox.io>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as UBports.
```

Comment:
"UBports" operating system here "UBports installer" somewhere else.

Reply:
probably neither. I guess Ubuntu Touch would be most correct